host/xtest/regression_6000.c

```diff
@@ -1911,6 +1915,7 @@ static void xtest_tee_test_6018_single(ADBG_Case_t *c, uint32_t storage_id)
 		uint8_t br[block_size];
 		uint32_t count = 0;
 
+		memset(br, 0, sizeof(br));
```

Comment:
How about using an initializer above instead?

Reply:
because the table size is considered as variable, our current toolchain maybe initialize properly:
`... error: variable-sized object may not be initialized`

Reply:
Fair enough